src/main/resources/schema-h2.sql

```diff
@@ -0,0 +1,86 @@
+CREATE SCHEMA IF NOT EXISTS lpvs;
```

Comment:
Let's use just one sql file, `database_dump.sql`.

Reply:
@m-rudyk  @v-kerimov  Agree with Taewan.

Reply:
schema-h2.sql and data-h2.sql files are necessary for correct initialization of an in-memory in the "singlescan" profile database and, from what I understand about Hibernate configurations, cannot be merged.
If the issue is that having too many sql files in one place looks cluttered we can, for example, move database_dump.sql to  root to hopefully make things clearer.

Reply:
Ok. Let's create a task related to DB dumps optimization and apply changes till the next release.